features/mbedtls/mbed_lib.json

```diff
@@ -0,0 +1,157 @@
+{
+    "name": "mbedtls",
+    "config": {
+        "MBED_CLOUD_CLIENT": {
+            "help": "Include support for ciphersuites required by the Mbed Cloud Client",
+            "value": 1
```

Comment:
Style nit - to fit the config system general style, the names should be lower-case with hyphens. (The build system will convert to upper-case with underscores when it turns them into defines).

Also `true` and `false` are better JSON.